ui/packages/security-idp-server/src/main/webapp/js/view/Login.view.js

```diff
@@ -42,6 +42,7 @@ define([
       }, 0)
     },
     logInEnter: function(e) {
+      this.$('#loginerrorup').hide()
```

Comment:
I think we should move this to the `logInUser` method.

Reply:
It essentially already calls this by going up to the ajax call in the before send when hitting enter. The point of this line here was to remove the error message if the username or password is changed at all (like adding or removing a letter), not just when enter is hit. Is this functionality we would want, or no?

Reply:
If we are clearing errors when they click the login button, we should be fine. I didn't realize  this was for any change to the inputs :ok_hand: